src/main/java/com/rocketden/main/dto/game/SubmissionResultDto.java

```diff
@@ -0,0 +1,17 @@
+package com.rocketden.main.dto.game;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class SubmissionResultDto {
+    // This class is mapped with the tester service's ResultDto class.
+    private String console;
+    private String userOutput;
+    private String error;
+    private String input;
+    private String correctOutput;
+    private boolean hidden;
```

Comment:
Note for future: if we go like a competition route we will prob have to remove some of these fields to prevent people inspecting element/cheating

Reply:
Lol hmm good point - we may want to have a sprint dedicated to all the security problems and bug fixes and whatnot. We could have like a UI sprint and a security / bug fix sprint 👍